### PR TITLE
Added dynamically changing review bars in ReviewBreakdown.jsx

### PR DESCRIPTION
**Pull request by ajpsyk**
Removed duplicate reviewsTotal function
Removed duplicate StarRating component in ReviewsAndRatings module
Made ReviewsAndRating div selectable by id

Review spread with 5 reviews
Screen Shot 2021-11-01 at 2 14 00 PM

Review spread with 255 reviews
Screen Shot 2021-11-01 at 2 17 21 PM

### CODE REVIEW

File: src/ratings-and-reviews/index.jsx [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
+    total += reviews[i];
+  }
+  return total;
+};
+
+
 const RatingsAndReviews = () => (
-  <div className="ratings-and-reviews">
-    <h2 className="r-and-r-header">Ratings & Reviews</h2>
+  <div id="ratings-and-reviews">
```
Comment (jnbooth) [inline src/ratings-and-reviews/index.jsx:24]: Thanks!

File: src/ratings-and-reviews/components/ReviewBreakdown.test.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  expect(document.querySelector('#progress-5-star')).toHaveAttribute('value', '100');
+
+  expect(document.querySelector('#progress-4-star')).toHaveAttribute('value', '70');
+
+  expect(document.querySelector('#progress-3-star')).toHaveAttribute('value', '50');
+
+  expect(document.querySelector('#progress-2-star')).toHaveAttribute('value', '10');
+
+  expect(document.querySelector('#progress-1-star')).toHaveAttribute('value', '25');
```
Comment (jnbooth) [inline src/ratings-and-reviews/components/ReviewBreakdown.test.jsx:18]: Very nice

File: src/ratings-and-reviews/components/Sort.jsx [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import {reviewsTotal} from '../index.jsx';
```
Comment (jnbooth) [inline src/ratings-and-reviews/components/Sort.jsx:2]: excellent PER5I6TENCE

**Review by jnbooth — APPROVED** [review]
It's good!

**Comment by ajpsyk** [comment]
[Create dynamically changing bars that represent number of reviews per star rating.](https://trello.com/c/jLUWrlnZ/66-create-dynamically-changing-bars-that-represent-number-of-reviews-per-star-rating)

**Comment by ajpsyk** [comment]
[Create dynamically changing bars that represent number of reviews per star rating.](https://trello.com/c/jLUWrlnZ/66-create-dynamically-changing-bars-that-represent-number-of-reviews-per-star-rating)

**Pull request merged**